### PR TITLE
Add payload lenght on CAN frame

### PR DESCRIPTION
**Pull request by Acuadros95**

### CODE REVIEW

**Review by pablogs9 — APPROVED** [review]

**Comment by Acuadros95** [comment]
@mergify backport foxy main

File: extra_sources/microros_transports/udp_transport_threadX.c [hunk 1]
```diff
@@ -39,8 +39,17 @@ void tx_application_define_user(void *first_unused_memory)
...
+	for(size_t i = 2; i < 6; i++)
+    {
+        uint32_t sum = 0;
+        size_t UniqueId_offset = (i-2)*4;
+
+        for (size_t j = 0; j < 4; j++)
+        {
+            sum += unique_id->unique_id_words[j+UniqueId_offset];
+        }
+
```
Comment (pablogs9) [inline extra_sources/microros_transports/udp_transport_threadX.c:51]: Remove this space

File: extra_sources/microros_transports/udp_transport_freeRTOS.c [hunk 2]
```diff
@@ -40,8 +40,17 @@ bool renesas_e2_transport_open(struct uxrCustomTransport * transport){
...
+    for(size_t i = 2; i < 6; i++)
+    {
+        uint32_t sum = 0;
+        size_t UniqueId_offset = (i-2)*4;
+
+        for (size_t j = 0; j < 4; j++)
+        {
+            sum += unique_id->unique_id_words[j+UniqueId_offset];
+        }
+
```
Comment (pablogs9) [inline extra_sources/microros_transports/udp_transport_freeRTOS.c:52]: Remove this space

**Review by pablogs9 — CHANGES_REQUESTED** [review]

File: extra_sources/microros_transports/udp_transport_freeRTOS.c (hunk 2 above)
Comment (Acuadros95) [reply extra_sources/microros_transports/udp_transport_freeRTOS.c]: eb8b459622b413d152eafe5f376e4aa5d6974620

File: extra_sources/microros_transports/udp_transport_threadX.c (hunk 1 above)
Comment (Acuadros95) [reply extra_sources/microros_transports/udp_transport_threadX.c]: eb8b459622b413d152eafe5f376e4aa5d6974620

**Review by pablogs9 — APPROVED** [review]

**Pull request merged**